src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -13,17 +13,18 @@ description: How to internationalize your Flutter app.
 {{site.alert.end}}
 
 If your app might be deployed to users who speak another language then
-you'll need to "internationalize" it. That means you'll need to write
-the app in a way that makes it possible to "localize" values like text
-and layouts for each language or "locale" that the app
+you'll need to internationalize it. That means you'll need to write
+the app in a way that makes it possible to localize values like text
+and layouts for each language or locale that the app
 supports. Flutter provides widgets and classes that help with
 internationalization and the Flutter libraries themselves are
 internationalized.
 
-The tutorial that follows is largely written in terms of the Flutter
-`MaterialApp` class, since most applications are written that way.
-Applications written in terms of the lower level `WidgetsApp` class
-can also be internationalized using the same classes and logic.
+This page covers concepts and workflows necessary to localize a
+Flutter application in terms of the `MaterialApp` class, as most apps
+are written that way. However, applications written in terms of the
+lower level `WidgetsApp` class can also be internationalized using
+the same classes and logic.
```

Comment:
What about Cupertino apps?

Reply:
Same way as `MaterialApp`. Updated it to say `MaterialApp`/`CupertinoApp`